app/controllers/topics_controller.rb

```diff
@@ -8,6 +8,38 @@ def index
     end
   end
 
+  def vote
+    @topic = Topic.find params[:id]
+    @topic.increment :votes
+    respond_to do |format|
+      if @topic.save 
```

Comment:
Haha! Good example. I'll switch to the `increment!` version so we are sure, however should we just skip validating for a positive integer or should i set the votes column to a positive integer column? For example by setting votes column to unsigned (only positive numbers)?